pkg/storage/bloom/v1/bloom.go

```diff
@@ -32,16 +32,14 @@ func (b *Bloom) Encode(enc *encoding.Encbuf) error {
 	return nil
 }
 
-func (b *Bloom) Decode(dec *encoding.Decbuf) error {
+type BloomQuerier struct {
```

Comment:
What's the reason for adding the `BloomQuerier` struct here?

Reply:
Removed with my last commit.